msal/build.gradle

```diff
@@ -91,7 +91,14 @@ android {
 
     lintOptions {
         abortOnError true
+        disable 'MissingPermission'
```

Comment:
@fadidurah why this?

Reply:
With this, the linting configuration matches that of common